src/components/ReplyComment/hooks/useReplyComment.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import Toast from 'react-native-simple-toast';
 import moment from 'moment';
-import {useNavigation} from '@react-navigation/core';
+import {useNavigation, useRoute} from '@react-navigation/core';
+import {Dimensions} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useRoute` and `Dimensions` from 'react-native' has been added. Ensure that these are necessary for the component's functionality and that they are used appropriately within the component.

```diff
+ import {useNavigation, useRoute} from '@react-navigation/core';
+ import {Dimensions} from 'react-native';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->